spek-samples/src/test/kotlin/org/jetbrains/spek/samples/subject/CalculatorTableSpec.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.spek.samples.subject
+
+import org.jetbrains.samples.Calculator
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+import org.jetbrains.spek.api.dsl.on
+import org.jetbrains.spek.table.example
+import org.jetbrains.spek.table.with
+import kotlin.test.assertEquals
+
+object CalculatorTableSpec : Spek({
+    describe("subtract") {
+        with(
+            example(4, 2),
```

Comment:
Seems less flexible if we add an `expected` param.
```kotlin
forEvery(
   example(param1_value, param2_value, expected = SomeException::class)
) { param1, param2 ->
   it("should throw SomeException") {
       expect({ some.foo(param1, param2) }).throws(expected)
   }
}
```

There will be cases that you only have one expected value/side-effect, it's to verbose if we include the expected value/side-effect for every example.
```kotlin
forEvery(
   example(param1_value, param2_value)
) { param1, param2 ->
   it("should throw SomeException") {
       expect({ some.foo(param1, param2) }).throws<SomeException>()
   }
}
```